tests/Snak.tests.js

```diff
@@ -8,11 +8,11 @@
 QUnit.module( 'wikibase.datamodel.Snak' );
 
 var testSets = [
-	[wb.datamodel.PropertyNoValueSnak, ['P1']],
-	[wb.datamodel.PropertyNoValueSnak, ['P2']],
-	[wb.datamodel.PropertySomeValueSnak, ['P1']],
-	[wb.datamodel.PropertyValueSnak, ['P1', new dv.StringValue( 'test' )]],
-	[wb.datamodel.PropertyValueSnak, ['P2', new dv.StringValue( 'test' )]]
+	[wb.datamodel.PropertyNoValueSnak, ['P1', undefined]],
```

Comment:
I think the two "undefined" can be omitted, but it doesn't hurt to have them.

Reply:
No, they’re necessary because the expected hash is the last array element (`array[array.length-1]`).

Reply:
Ah, thanks! I see it now.